algebra-utils/src/msm/variable_base.rs

```diff
@@ -1,9 +1,17 @@
-use crate::{AffineCurve, BigInteger, Field, FpParameters, PrimeField, ProjectiveCurve};
+use algebra::{AffineCurve, BigInteger, Field, FpParameters, PrimeField, ProjectiveCurve};
 use rayon::prelude::*;
 
+#[cfg(feature = "gpu")]
+use algebra_kernels::msm::{get_cpu_utilization, get_kernels, get_gpu_min_length};
+#[cfg(feature = "gpu")]
+use algebra_cl_gen::gpu::GPUError;
+#[cfg(feature = "gpu")]
+use crossbeam::thread;
```

Comment:
In the future we should consider using everywhere either rayon or crossbeam, it's a little bit ugly to keep both.

Reply:
Crossbeam is removed, now only rayon is used